src/main/java/com/game/Battleship/controllers/BoardController.java

```diff
@@ -1,17 +1,52 @@
 package com.game.Battleship.controllers;
-
 import com.game.Battleship.models.Board;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RestController;
+import com.game.Battleship.models.Ship;
+import com.game.Battleship.repository.BoardRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+import java.util.List;
+import java.util.Map;
 
 @RestController
 public class BoardController {
 
-    @GetMapping("/board")
-    public Board getBoard(){
+    @Autowired
+    private BoardRepository boardRepository;
+
+    @GetMapping("/boards")
+    public List<Board> getAllBoards(){
+
+        return boardRepository.allBoards();
+
+    }
+
+    //json object example: {"row": 0, "column": 1, "shipType": "Cruiser", "shipSize": 3, "isHorizontal": false }
```

Comment:
Is this comment for documentation? You want to minimize comments in your codebase because it's another thing that has to be updated if something changes. I'm not saying remove this comment, just choose your battles when you leave comments. Your code and tests should do the explaining, but if it's absolutely necessary to explain something for a future developer or yourself 6 months down the road then it's appropriate to leave comments. 😃

Reply:
Yes, I leave it for documentation for now. This is the Json object form for now but I will need to change it and change some code too because Lane has a different form.